From 4be4dd116dda808bf1c9184507d0932ceba1193e Mon Sep 17 00:00:00 2001
From: Ya Tang <yatang@qti.qualcomm.com>
Date: Mon, 8 Sep 2025 13:32:33 +0800
Subject: [PATCH] Remove LOADER_ENABLE_LINUX_SORT Macro Definition

Eliminate the LOADER_ENABLE_LINUX_SORT compile-time macro
to disable the Vulkan Loader's physical device reordering
logic on Linux platforms.

Upstream-Status: Inappropriate [oe specific]
Change-Id: I8dcf029436723024591acc3367fea10c64578664
Signed-off-by: Ya Tang <yatang@qti.qualcomm.com>
---
 loader/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/loader/CMakeLists.txt b/loader/CMakeLists.txt
index df269befd..bc40cc8a5 100644
--- a/loader/CMakeLists.txt
+++ b/loader/CMakeLists.txt
@@ -108,7 +108,7 @@ if(WIN32)
     list(APPEND NORMAL_LOADER_SRCS loader_windows.c dirent_on_windows.c)
 elseif(CMAKE_SYSTEM_NAME MATCHES "Linux|BSD|DragonFly|GNU")
     list(APPEND NORMAL_LOADER_SRCS loader_linux.c)
-    target_compile_definitions(loader_specific_options INTERFACE LOADER_ENABLE_LINUX_SORT)
+    #target_compile_definitions(loader_specific_options INTERFACE LOADER_ENABLE_LINUX_SORT)
 endif()
 
 if ("${CMAKE_OSX_ARCHITECTURES}" MATCHES ";")
-- 
The Qualcomm Innovation Center, Inc. is a member of the Code Aurora Forum,
a Linux Foundation Collaborative Project

